### PR TITLE
add dark theme to PassphraseChangeActivity

### PR DESCRIPTION
**Pull request by McLoo**
also adding DynamicLanguage to fix ActionBar language changes

Though I'm not convinced of that shadow stuff, it leave it be - Moxie surely had reasons for that...
(Someone) might throw it away when fixing the rest of the registration activities.

### CODE REVIEW

**Comment by McLoo** [comment]
unbenannt

File: res/values/styles.xml [hunk 1]
```diff
@@ -108,11 +108,11 @@
         <item name="android:textSize">12.0sp</item>
         <item name="android:typeface">sans</item>
         <item name="android:textStyle">normal</item>
-        <item name="android:textColor">#ff808080</item>
+        <item name="android:textColor">?android:attr/textColorSecondary</item>
```
Comment (mcginty) [inline res/values/styles.xml]: we should own this attribute, ROMs could change it leading to a bad visual experience

Comment (McLoo) [inline res/values/styles.xml]: so `<item name="android:textColorSecondary">@color/text_color_secondary_dark_theme</item>` would do the trick? [themes.xml#L140](https://github.com/McLoo/TextSecure/blob/fix_2625_theming/res/values/themes.xml#L140)

Or are you more into for having a new registration_text_color attribute?

**Comment by rhodey** [comment]
please test this, fickle bot

**Comment by mcginty** [comment]
closing in favor of https://github.com/WhisperSystems/Signal-Android/pull/4570 which includes your commit from this PR

**Comment by McLoo** [comment]
finally, thanks :)

**Pull request closed without merging**